linked_list/linked_list.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import print_function  # For Python 2
+
+
+class node(object):
```

Comment:
Class names in Python usually start with an uppercase character. 
https://www.python.org/dev/peps/pep-0008/#class-names